app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -232,14 +217,10 @@ public void onDetachedFromRecyclerView(RecyclerView recyclerView) {
                 return new PieChartViewHolder(view, clickListener);
             case SEARCH_RESULT:
                 view = inflater.inflate(R.layout.search_list, viewGroup, false);
-                SearchResultsListHolder searchResultsListHolder = new SearchResultsListHolder(view, clickListener);
-                searchResultsListHolder.recyclerView.addItemDecoration(new ConstraintsHelper(6, currContext));
-                return new SearchResultsListHolder(view, clickListener);
+                return new SearchResultsListHolder(view,clickListener);
             case WEB_SEARCH:
                 view = inflater.inflate(R.layout.search_list, viewGroup, false);
-                SearchResultsListHolder webResultsListHolder = new SearchResultsListHolder(view, clickListener);
-                webResultsListHolder.recyclerView.addItemDecoration(new ConstraintsHelper(6, currContext));
-                return new SearchResultsListHolder(view, clickListener);
+                return new SearchResultsListHolder(view,clickListener);
```

Comment:
I solved it in other copy of project and by mistake I copied everything and pasted here. I deleted by mistake I will correct this.